src/components/atoms/inputField/Component.tsx

```diff
@@ -0,0 +1,26 @@
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+
+
+export const InputField = ():JSX.Element => {
+    const style={
+        'background': '#F3F1F1',
+        // '&:hover' : {
```

Comment:
are we still using these commented code? if not maybe delete?

Reply:
No im not using these commented code. I am going to delete it asap.